components/Card.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+
+export default function card({ voc }) {
```

Comment:
It's generally a good practice to start the names of React components with a capital letter. So instead of `card`, consider renaming it to `Card`.